quintilesims/layer0
# 232: LoadBalancers as new benchmark parameter, more baseline benchmark tests

**What does this pull request do?**
Merging in work @tlake and I have combined to do into our shared feature branch.
Implemented Load Balancers as a parameterized value that can be provided for a benchmark test.
Added several baseline benchmark tests

**How should this be tested?**
The benchmarks should run by running `make test` from the tests/stress directory. The README in the tests/ directory provide some notes for consideration when preparing to run the stress tests.

**Checklist**
~- [ ] Unit tests~
~- [ ] Smoke tests (if applicable)~
~- [ ] System tests (if applicable)~
